controllers/prof_controller.go

```diff
@@ -0,0 +1,81 @@
+package controllers
```

Comment:
Rename file about profile pages to "profile_controller.go" for example. This is to avoid confusion.